InternetShop/InternetShop/ViewModels/Basket/BasketViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using InternetShop.ViewModels.Product;
+using IS.BLL.Models;
+
+namespace InternetShop.ViewModels.Basket
+{
+    public class BasketViewModel
+    {
+#nullable disable
+        public int Id { get; set; }
+        public string UserId { get; set; }
+        public ICollection<ProductViewModel> Products { get; set; }
```

Comment:
Maybe enable nullable?
Because user doen't have product in basket when he visit site the first time